WORKSPACE

```diff
@@ -83,6 +83,16 @@ pip_parse(
     requirements_windows = "//tools/publish:requirements_windows.txt",
 )
 
-load("@publish_deps//:requirements.bzl", "install_deps")
+load("@publish_deps//:requirements.bzl", publish_install_deps = "install_deps")
 
-install_deps()
+publish_install_deps()
+
+pip_parse(
+    name = "examples_wheel_deps",
```

Comment:
Shouldn't this be in the example's workspace file...?

Reply:
That example doesn't have its own WORKSPACE.